Release.toml

```
@@ -445,3 +445,6 @@ version = "1.50.0"
 "(1.49.0, 1.50.0)" = [
     "migrate_v1.50.0_kubernetes-reserved-pid-settings.lz4",
 ]
+"(1.50.0, 1.51.0)" = [
+    "migrate_v1.51.0_kubernetes-ecr-credential-providers-pattern.lz4",
```

```suggestion
    "migrate_v1.51.0_kubernetes-ecr-credential-provider-patterns.lz4",
```

Not sure if this is a typo or if I'm missing something

---

good catch, updated